algebird-core/src/main/scala/com/twitter/algebird/MomentsGroup.scala

```diff
@@ -53,7 +53,55 @@ sealed class Moments(val m0D: Double, val m1: Double, val m2: Double, val m3: Do
     math.sqrt(m0D) * m3 / math.pow(m2, 1.5)
 
   def kurtosis: Double =
-    m0D * m4 / math.pow(m2, 2) - 3
+    m0D * m4 / (m2 * m2) - 3
+
+  /**
+   * Combines this instance with another [[Moments]] instance.
+   * @param b
+   *   the other instance
+   * @return
+   *   a [[Moments]] instances representing the combined moments of this
+   *   instance and and `b`
+   */
+  def +(b: Moments): Moments = Moments.momentsMonoid.plus(this, b)
+
+  /**
+   * Returns a new [[Moments]] instance generated by merging in the new
+   * observation `b`.
+   * @param b
+   *   a new observation
+   * @return
+   *   a [[Moments]] instance representing the combined moments of this
+   *   instance and and `b`.
+   */
+  def +(b: Double): Moments = {
+    val n = m0D + 1
+    val delta = b - mean
+    val delta_n = delta / n
+    val delta_n2 = delta_n * delta_n
+    val term1 = delta * delta_n * m0D
+
+    val meanCombined = Moments.getCombinedMeanDouble(m0D, mean, 1.0, b)
+    val m2combined = m2 + term1
+    val m3combined = m3 + term1 * delta_n * (n - 2) - 3 * delta_n * m2
+    val m4combined = m4 + term1 * delta_n2 * (n * n - 3 * n + 3) +
+      6 * delta_n2 * m2 - 4 * delta_n * m3
+
+    new Moments(n, meanCombined, m2combined, m3combined, m4combined)
+  }
+
+  /**
+    * Returns a [[Fold]] instance that uses `+` to accumulate deltas into this
+    * [[Moments]] instance.
+    */
+  def fold: Fold[Double, Moments] =
```

Comment:
This is the style we used in `ExpHist`, where the fold is an instance method that lets you fold into THIS `Moments`, implemented by

```scala
(state: Moments.MomentsState) => this + state.toMoments
```

Let me know if you want to just have an empty fold on `Moments`.